gsw_check_functions.c

```diff
@@ -96,6 +98,32 @@ double          tf_poly[cast_m*cast_n];
 double          h[cast_m*cast_n];
 double          z[cast_m*cast_n];
 
+int assert_equal(int actual, int expected, const char *test_name) {
+    if (actual == expected) {
+        printf("%s: ............................... passed\n", test_name);
+    } else {
+        printf("%s: Failed (Expected: %d, Actual: %d)\n", test_name, expected, actual);
+        return gsw_error_flag=1;
+    }
+}
+
+
+void test_infunnel() {
+    // Inside the funnel.
+    int result1 = gsw_infunnel(35.0, 5.0, 400);
```

Comment:
I would put a single declaration "int result;" after the opening brace, and then use that in place of the numbered versions.  Nothing is gained by using different variables.  Although the most modern C allows variable declarations in other locations, most C still follows the earlier pattern of keeping variable declarations restricted to being after an opening brace (of the function, or of a scope within the function).

Reply:
Thanks! Just learned something new. Learning C as I go here 😬